src/main/java/com/github/forax/umldoc/Main.java

```diff
@@ -30,8 +30,7 @@ public static void main(String[] args) throws IOException {
             var reader = Files.newBufferedReader(Path.of(args[1]));
             var writer = Files.newBufferedWriter(Path.of("resultFile.md"))
     ) {
-      var config = new HashMap<String, CommandLineParser>();
-      config.put("mermaid", new MermaidCmdLineParser());
+      Map<String, CommandLineParser> config = Map.of("mermaid", new MermaidCmdLineParser());
```

Comment:
please use 'var'

Reply:
can't use var because it will create a Map<String, MermaidCmdLineParser> instead of a Map<String, CommandLineParser> (CommandLineParser being the interface that MermaidCmdLineParser implements)

Reply:
`var config = Map.<WhatEver, WhatEver>of(...);`